### PR TITLE
Update guidance document for the github merge workflow

### PR DESCRIPTION
**Pull request by mwaskom**

### CODE REVIEW

File: tutorials/reviewing-tutorials.md [hunk 1]
```diff
@@ -32,26 +32,38 @@ See [Content Review Sign-Up](https://docs.google.com/spreadsheets/d/1LtLEk0H7gkR
...
+  - For the CI workflow to run properly, the PR must be made from a feature branch on the main repository, not a fork.
+2. Notebooks are required to have a sequential execution history on a fresh kernel. This will be automatically checked as part of CI. Before you push, execute the entire notebook (`Runtime -> Restart and run all` in the Colab menu) and make sure it completes without any errors. Then save the notebook.
+3. Push the notebook from Colab to Github:
+  - `File -> Save a copy in Github` in the Colab menu
+  - Select the course repository: (`NeuromatchAcademy/course-content`)
+  - Select your feature branch: (e.g. `WxDy`)
+  - Write out the complete file path, using the standardized template:
+    - `tutorials/W{x}D{y}-{Topic}/WxDy_Tutorial{i}.ipynb`
+    - Replace `{x}`, `{y}`, `{Topic}` and `{i}`
+    - See the [tutorials directory](https://github.com/NeuromatchAcademy/course-content/tree/master/tutorials) for a list names to use for `{Topic}`
```
Comment (patrickmineault) [inline tutorials/reviewing-tutorials.md:54]:
```suggestion
    - See the [tutorials directory](https://github.com/NeuromatchAcademy/course-content/tree/master/tutorials) for a list of names to use for `{Topic}`
```

File: tutorials/reviewing-tutorials.md [hunk 2]
```diff
@@ -32,26 +32,38 @@ See [Content Review Sign-Up](https://docs.google.com/spreadsheets/d/1LtLEk0H7gkR
...
+    - Note: the directory name uses a dash (`-`), but the filename uses an underscore (`_`). This is regrettable, but please maintain the convention.
```
Comment (patrickmineault) [inline tutorials/reviewing-tutorials.md:55]: We could also just rename everything.

File: tutorials/reviewing-tutorials.md [hunk 3]
```diff
@@ -32,26 +32,38 @@ See [Content Review Sign-Up](https://docs.google.com/spreadsheets/d/1LtLEk0H7gkR
...
+    - An example filename will look like this: `tutorials/W1D5-DimensionalityReduction/W1D5_Tutorial3.ipynb`
+    - Don't copy the filename from a rich text display (e.g. the Github UI), which seems to produce duplicate files.
```
Comment (patrickmineault) [inline tutorials/reviewing-tutorials.md:57]:
```suggestion
    - Don't copy the filename from a rich text display (e.g. the Github UI), which sometimes copies hidden Unicode characters. Navigate to the Github location in your browser and copy the path from the URL.
```

File: tutorials/reviewing-tutorials.md [hunk 4]
```diff
@@ -32,26 +32,38 @@ See [Content Review Sign-Up](https://docs.google.com/spreadsheets/d/1LtLEk0H7gkR
...
+  - Make sure that "Include a link to Colaboratory" remains checked.
+  - **Important**: Double-check that the file path is correct before pushing. Fixing incorrect names is a substantial pain, because multiple derivative files will be created using the name you push.
+4. Repeat Step 3 with the other tutorials for that day.
+5. On Github, open a Pull Request from your feature branch onto `master`.
+6. Opening the PR will trigger the notebook CI workflow. This will (1) run QC checks on the notebooks and (2) create student versions and derivative static files (solution images and scripts).
+  - If the checks fail, click through to the Github Action log and try to figure out what's wrong (or ask for help on Slack).
```
Comment (patrickmineault) [inline tutorials/reviewing-tutorials.md:63]:
```suggestion
  - If the checks fail, click through to the Github Action log and try to figure out what's wrong (or ask for help on Slack).
  - Fix the problems and commit again. The CI will run again. 
```

Comment (mwaskom) [reply tutorials/reviewing-tutorials.md:62]: Done.

File: tutorials/reviewing-tutorials.md (hunk 3 above)
Comment (mwaskom) [reply tutorials/reviewing-tutorials.md]: Done.

File: tutorials/reviewing-tutorials.md (hunk 2 above)
Comment (mwaskom) [reply tutorials/reviewing-tutorials.md]: Done.

File: tutorials/reviewing-tutorials.md (hunk 1 above)
Comment (mwaskom) [reply tutorials/reviewing-tutorials.md]: Done.

**Pull request merged**